test/07-openqa-label-known-issues.t

```diff
@@ -42,8 +43,16 @@ out=''
 hxnormalize() {
     cat "$2"
 }
+
 hxselect() {
-    cat -
+    if [ -p /dev/stdin ]; then
```

Comment:
Please check if it is still necessary at all to overwrite hxnormalize and hxselect here.

Reply:
they seems to work. What have changed? I think the reason I did this was that the tools were not available before

Reply:
what for? tests are passing for me locally without those functions

Reply:
html-xml-utils is now installed in CI.
Otherwise a comment in the test would have been useful to tell people who don't know it and forgetful people like me like "These commands are not available in our CI, mocking"